user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleInformationMain.java

```diff
@@ -264,8 +265,10 @@ private void onEditBatchClicked(EditBatchEvent editBatchEvent) {
         editBatchEvent.batchPreview().batchId()).stream().toList();
     var experimentalGroups = experimentInformationService.experimentalGroupsFor(
         context.experimentId().get());
-    List<SampleBatchInformationSpreadsheet.SampleInfo> sampleInfos = samples.stream()
-        .map(sample -> convertSampleToSampleInfo(sample, experimentalGroups)).toList();
+    // need to create mutable list to order samples
+    List<SampleBatchInformationSpreadsheet.SampleInfo> sampleInfos = new ArrayList<>(samples.stream()
+        .map(sample -> convertSampleToSampleInfo(sample, experimentalGroups)).toList());
```

Comment:
```suggestion
    List<SampleBatchInformationSpreadsheet.SampleInfo> sampleInfos = samples.stream()
        .map(sample -> convertSampleToSampleInfo(sample, experimentalGroups))
        .collect(Collectors.toCollection(ArrayList::new));
```

Reply:
du kannst auch den stream schon sorten
```suggestion
    var sampleInfos = samples.stream()
        .map(sample -> convertSampleToSampleInfo(sample, experimentalGroups))
        .sorted(Comparator.comparing(o -> o.getSampleCode().code())
        .collect(Collectors.toCollection(ArrayList::new));
```